frontend/src/pages/answer/Answer.jsx

```diff
@@ -111,16 +114,37 @@ export default function Answer() {
     validateAndInitializeMessage(answerResponse);
   }
 
+  /**
+   * Get metadata of answer to see if current user owns it
+   */
+  async function checkIfAnswerOwned() {
+    let accessToken;
+    if (isAuthenticated) {
+      try {
+        accessToken = await getAccessTokenSilently();
+      } catch (err) {
+        displayAlert('error', `Failed to validate user. Error: ${err}`);
+      }
+    }
+    const answerResponse = await API.cache.getAnswer(answer_id, accessToken);
```

Comment:
Is determining whether the answer is owned really the only reason we need the metadata? It seems odd that we make the API call here, grab `.owned`, and throw away the rest.

Reply:
Yes. This call is very quick, as we only get the metadata of the answer, not the actual message. This is so we can disable the "Delete Answer" button so non-answer owners don't think they can even delete an answer they don't own.

Reply:
Add documentation in df5308d6a1e26386aba4ef0b897106f9e0a2c2d2

Reply:
Let's keep an eye on this. I can imagine us wanting a variety of different bits of metadata, and we don't want to end up making the same call repeatedly.